src/ring_theory/graded_algebra/homogeneous_ideal.lean

```diff
@@ -61,6 +61,18 @@ abbreviation homogeneous_ideal : Type* := { I : ideal A // I.is_homogeneous 𝒜
 instance : has_mem A (homogeneous_ideal 𝒜) :=
 { mem := λ r I, r ∈ (I : ideal A) }
```

Comment:
According to https://leanprover-community.github.io/mathlib_docs/data/set_like/basic.html you should remove these lines, as they're implied by the other lines